tests/flex_opt/test_reinforce_grid.py

```diff
@@ -58,12 +58,10 @@ def test_reinforce_grid(self):
         # test reduced analysis
         res_reduced = reinforce_grid(
             edisgo=copy.deepcopy(self.edisgo),
-            timesteps_pfa="reduced_analysis",
-            num_steps_loading=4,
-        )
-        assert_frame_equal(
-            res_reduced.equipment_changes, results_dict[None].equipment_changes
+            reduced_analysis=True,
+            num_steps_loading=2,
         )
+        assert len(res_reduced.i_res) == 2
```

Comment:
Alot more is for the reinforcement tested in the [EDisGo class tests](https://github.com/openego/eDisGo/blob/wp_flex_changed_reinforcement/tests/test_edisgo.py#L432-L558). At this point I just wanted to check, whether the usage of two time steps is followed through throughout the reinforcement. Do you still think more tests need to be added or is it okay?

Reply:
Ah then it is probably okay. It seems like it is never tested for specific values (e.g. the sum of reinforcement costs in a specific case). Is that on purpose? Would we not want to be able to detect if something changes with the values as well, not just the number of reinforced elements?

Reply:
It's not on purpose and could be added some time. I would rather test the content of equipment changes then, because the costs function test already tests the derivation of the costs from the equipment changes. I'll add a ToDo for now.